tests/Unit/Campaigns/Migrations/MigrateFormsToCampaignFormsTest.php

```diff
@@ -19,6 +20,8 @@ final class MigrateFormsToCampaignFormsTest extends TestCase
 
     /**
      * @unreleased
+     *
+     * @throws Exception
      */
     public function testCreatesParentCampaignForDonationForm()
```

Comment:
@glaubersilva can you add a new test to cover V2 forms that are not upgraded?

Reply:
Ok, I think I better understand what you said after I wrote my last comment! 😅

Resolved in eac91275f88f3f74d95e4dc43b4bde48c1f28472